mySQL-example/MySQL.go

```diff
@@ -0,0 +1,183 @@
+package scripts
+
+import (
+	"database/sql"
+	"fmt"
+	"time"
+	//MySQL driver
+	_ "github.com/go-sql-driver/mysql"
+	//convert to strings
+	// import of application related stuff
+	"git.monkey-works.de/scripting/api"
+	"monkey-works.de/model"
+)
+
+//The endpoint of our web service we want to connect to
+//Change this to your endpoint
+const URL string = "tcp(127.0.0.1:3306)/schema" //tcp(yourHost_IP:Port)/Schema_name
```

Comment:
May you change that to one big const block? E.g.:
`const (
  ...
)` See: https://tour.golang.org/basics/16. This should help the formatter and increase readibility